idl/storage_service.idl.hh

```diff
@@ -60,7 +60,8 @@ struct raft_topology_snapshot {
 struct raft_topology_pull_params {};
 
 struct raft_snapshot {
-    utils::chunked_vector<canonical_mutation> mutations;
+    utils::chunked_vector<canonical_mutation> auth_mutations;
+    std::vector<canonical_mutation> service_levels_mutations;
 };
```

Comment:
The `raft_snapshot` RPC is supposed to be generic, i.e. you ask another node for mutations from given tables. We shouldn't add fields like this, just add the IDs of SL-related tables to the list in the request.